pandas/tests/frame/test_dtypes.py

```diff
@@ -149,6 +149,10 @@ def test_select_dtypes_include(self):
         ei = df[['k']]
         assert_frame_equal(ri, ei)
 
```

Comment:
add as a separate test, try both with include and exclude as scalaras (separately)